selenium_tests/test_storage.py

```diff
@@ -15,13 +15,14 @@ def __init__(self, methods):
     def test_if_upload_successfully(self):
         """ Test that upload file that assign correct value to input section """
         inputs = [
-            {'name': 'voltage', 'value': '5'},
-            {'name': 'length', 'value': '5'},
-            {'name': 'r0', 'value': '5'},
-            {'name': 'ra', 'value': '5'},
-            {'name': 'x', 'value': '5'},
-            {'name': 'force', 'value': '5'},
-            {'name': 'awg', 'value': '5'},
+            {'name': 'voltage', 'value': '5', 'unit': ''},
+            {'name': 'length', 'value': '5','unit': 'mm'},
+            {'name': 'r0', 'value': '5', 'unit': 'mm'},
+            {'name': 'ra', 'value': '5', 'unit': 'mm'},
+            {'name': 'x', 'value': '5', 'unit': 'mm'},
+            {'name': 'force', 'value': '5', 'unit': 'N'},
+            {'name': 'awg', 'value': '5', 'unit': ''},
+            {'name': 'relative_permeability', 'value': '5'},
         ]
         path = str(pathlib.Path().absolute()) + '/selenium_tests/'
```

Comment:
Can we move this to something similar to URL?  It is used multiple time in this file. Can say something like
```python3
PATH = str(pathlib.Path().absolute())
```